### PR TITLE
homepage and wishlist

### PR DESCRIPTION
**Pull request by anasm11**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      <div class='sale-section-text'>
+        <h1>Clearance sale 20-30% off.</h1>
+        <h2>Fresh Arrivals</h2>
+        <h2>Starting from ₹399</h2>
+        <h2>Buy Now</h2>
+      </div>
+    </div>
+
+    <div class='collection-1'>
+      <img src="./assets/shirt2.jpg" >
```
Comment (pradipgharge) [inline index.html:59]: image missing alt tag here

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+.category-container img{
+    height:14rem;
+    width:14rem;
+}
+
+.sale-section{
+    grid-column: 1/-1;
+    width:100%;
+    height:400px;
```
Comment (pradipgharge) [inline styles.css]: use rems for length units

**Review by pradipgharge — APPROVED** [review]
Add alt tags to all images. Rest looks good to merge

File: index.html (hunk 1 above)
Comment (anasm11) [reply index.html:59]: ok...i'll do that

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+}
+
+.sale-section>img{
+object-fit:fill;
+height:100%;
+}
+
+.collection-1{
+    grid-column:1;
+    height:200px;
```
Comment (Mohitk391) [inline styles.css]: Suggestion: You should use rem instead of pixels of responsive code

**Review by Mohitk391 — APPROVED** [review]
You can work on the suggestions, other than that your code and UI looks clean, consistent.
Looks good to merge..you can merge it

**Pull request merged**